Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN VERSION=${version} PLUGINS=${plugins} ENABLE_TELEMETRY=${enable_telemetry} /
 #
 # Final stage
 #
-FROM alpine:3.8
+FROM alpine
```

Comment:
Hi, kindly lock it to an alpine version.

Reply:
Fixed Alpine Image to 3.10